docs/hazmat/primitives/asymmetric/dh.rst

```diff
@@ -6,6 +6,28 @@ Diffie-Hellman key exchange
 .. currentmodule:: cryptography.hazmat.primitives.asymmetric.dh
 
 
+Exchange Algorithm
+~~~~~~~~~~~~~~~~~~
+
+    For most applications the ``shared_key`` should be passed to a key
+    derivation function.
+
+    .. doctest::
+
+        >>> from cryptography.hazmat.backends import default_backend
+        >>> from cryptography.hazmat.primitives.asymmetric import dh
+        >>> parameters = dh.generate_parameters(2, 512, default_backend())
```

Comment:
Pass `2` and `512` as keyword arguments, and replace 512 with 2048 -- examples should always use best practices